src/test/java/net/sf/jabref/importer/fileformat/InspecImportTest.java

```diff
@@ -0,0 +1,121 @@
+
+public class InspecImportTest {
+
+    private InspecImporter inspecImp;
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+        this.inspecImp = new InspecImporter();
+    }
+
+    @Test
+    public void testIsRecognizedFormatAccept() throws IOException {
+        List<String> testList = Arrays.asList("InspecImportTest.txt", "InspecImportTest2.txt");
+        for (String str : testList) {
+            try (InputStream inStream = InspecImportTest.class.getResourceAsStream(str)) {
+                assertTrue(inspecImp.isRecognizedFormat(inStream));
+            }
+        }
+    }
+
+    @Test
+    public void testIsRecognizedFormatReject() throws IOException {
+        List<String> testList = Arrays.asList("CopacImporterTest1.txt", "CopacImporterTest2.txt",
+                "IEEEImport1.txt", "IsiImporterTest1.isi", "IsiImporterTestInspec.isi", "IsiImporterTestWOS.isi",
+                "IsiImporterTestMedline.isi", "RisImporterTest1.ris", "InspecImportTestFalse.txt");
+
+        for (String str : testList) {
+            try (InputStream inStream = InspecImportTest.class.getResourceAsStream(str)) {
+                assertFalse(inspecImp.isRecognizedFormat(inStream));
+            }
+        }
+    }
+
+    @Test
+    public void testCompleteBibtexEntryOnJournalPaperImport() throws IOException {
+        
+        BibEntry shouldBeEntry = new BibEntry();
+
+        shouldBeEntry.setType("article");
+        shouldBeEntry.setField("title", "The SIS project : software reuse with a natural language approach");
+        shouldBeEntry.setField("author", "Prechelt, Lutz");
+        shouldBeEntry.setField("year", "1992");
+        shouldBeEntry.setField("abstract", "Abstrakt");
+        shouldBeEntry.setField("keywords", "key");
+        shouldBeEntry.setField("journal", "10000");
+        shouldBeEntry.setField("pages", "20");
+        shouldBeEntry.setField("volume", "19");
+
+        try (InputStream inStream = InspecImportTest.class.getResourceAsStream("InspecImportTest2.txt")) {
+            List<BibEntry> entries = inspecImp.importEntries(inStream, new OutputPrinterToNull());
+
+            assertEquals(1, entries.size());
+            BibEntry entry = entries.get(0);
+
+            BibtexEntryAssert.assertEquals(shouldBeEntry, entry);
+
+        }
+    }
+
+    @Test
+    public void importConferencePaperGivesInproceedings() throws IOException {
+        String testInput = "Record.*INSPEC.*\n" +
+                "\n" +
+                "RT ~ Conference-Paper";
+        try (InputStream inStream = new ByteArrayInputStream(testInput.getBytes())) {
+            List<BibEntry> entries = inspecImp.importEntries(inStream, new OutputPrinterToNull());
+            inStream.close();
+            assertEquals(1, entries.size());
+            BibEntry entry = entries.get(0);
+            assertEquals(BibtexEntryTypes.INPROCEEDINGS.getName().toLowerCase(), entry.getType());
```

Comment:
If i try to solve it with `Arrays.asList` or with `Collections.singletonList`, I get an error from the assertion because I try to compare a list of BibEntry with a BibEntry.